core/src/executor/execute.rs

```diff
@@ -256,6 +257,29 @@ async fn execute_inner<T: GStore + GStoreMut>(
 
             Ok(Payload::ShowColumns(output))
         }
+        Statement::ExplainTable { table_name } => {
+            let Schema { column_defs, .. } = storage
+                .fetch_schema(table_name)
+                .await?
+                .ok_or_else(|| ExecuteError::TableNotFound(table_name.to_owned()))?;
+
+            let output: Vec<(String, DataType, bool, String, String, String)> = column_defs
```

Comment:
It would be better to wrap verbose tuple with struct like..
```rs
struct ExplainTableRow {
   name: String,
   data_type: DataType,
   nullable: bool,
   key: String,
   default: String,
}
```